### ISSUE TITLE
Instructor: edit rubric question: tweak UI to prevent misunderstanding of weights 

### ISSUE THREAD

**Issue by damithc** [TEAMMATES/teammates#8824]
v1.5.1

Current: Instructors can set rubric question as shown below, thinking that it would make the options _weighted_
![image](https://user-images.githubusercontent.com/1673303/39968075-c03bb29e-56f9-11e8-925b-4518ed341966.png)

whereas the expected correct setting is as follows:
![image](https://user-images.githubusercontent.com/1673303/39968082-ebf14b4c-56f9-11e8-8ba6-5ea082412d10.png)

I have come across two users doing this mistake in the last few weeks. We should tweak the UI to prevent such mistakes. 

One possible approach (other suggestions welcome): instead of a tick box, use a drop down with options `Not weighted` and `weighted` so that is it clear the choices are not weighted.

**Comment by sukanta-27** [comment]
@damithc Sir, just to be sure, the instructors are setting the weights in the `choices` boxes without checking the checkbox right?

**Comment by damithc** [comment]
> @damithc Sir, just to be sure, the instructors are setting the weights in the choices boxes without checking the checkbox right?

Yes.

**Comment by damithc** [comment]
The good news is weights can be added even later, without deleting the existing responses :-)

**Comment by mg14777** [comment]
@damithc Sir, I will work on this

**Comment by sukanta-27** [comment]
![screenshot 489](https://user-images.githubusercontent.com/10681755/40012475-7029e224-57c8-11e8-92a2-86a6bb061384.png)

@damithc Sir, does this UI look good? I haven't modified the Javascript yet, just the layout.

**Comment by damithc** [comment]
> @damithc Sir, does this UI look good? I haven't modified the Javascript yet, just the layout.

Looks good. May be no need to bold?
How hard is it to take out of that 0,0 cell and put above the grid? That's where we usually show settings.

**Comment by damithc** [comment]
> P.S: Sir, just wondering that if we have to take out of the 0,0 cell and put the option above the grid, why are we not using the checkbox and renaming it to something like Add weights? Isn't a checkbox better suited for the situation (where we have kind of a true, false situation)?

Yes, in that case a check box might work. The label should be indicate the true state of the option 

e.g., `[ ] choices are weighted`
